File: entities/difficulty.py
class Difficulty():
    def __init__(self, easy=0, medium=0, hard=0, deadly=0):
        self.easy = easy
        self.medium = medium
        self.hard = hard
        self.deadly = deadly

    def __str__(self):
        return f"easy={self.easy}, medium={self.medium}, hard={self.hard}, deadly={self.deadly}"

File: TrainDataGen/Entities/Difficulty.cs
namespace TrainDataGen.Entities;

public class Difficulty
{
    public int Easy { get; set; }
    public int Medium { get; set; }
    public int Hard { get; set; }
    public int Deadly { get; set; }

    public Difficulty(int easy = 0, int medium = 0, int hard = 0, int deadly = 0)
    {
        Easy = easy;
        Medium = medium;
        Hard = hard;
        Deadly = deadly;
    }

    public override string ToString()
    {
        return $"easy={Easy}, medium={Medium}, hard={Hard}, deadly={Deadly}";
    }
}